### ISSUE TITLE
HTTP Proxy service failed to start

### ISSUE THREAD

**Issue by dingwen07** [AdguardTeam/AdguardForMac#1453]
### Please answer the following questions for yourself before submitting an issue

- [X] Filters were updated before reproducing an issue
- [X] I checked the [knowledge base](https://adguard.com/kb/) and found no answer
- [X] I checked to make sure that this issue has not already been filed

### AdGuard version

2.15.0.1723

### OS version

macOS Sonoma 14.5; Darwin Kernel Version 23.5.0

### Ad Blocking

AdGuard Base filter

### Privacy

AdGuard URL Tracking filter

### Social

AdGuard Social Media filter

### Which DNS server do you use?

DNS protection disabled

### DNS protocol

Regular

### Issue Details

Steps to reproduce:
1. Disable automatic network filtering
2. Make sure port 8123 (or any other port) is not used
3. Enable HTTP Proxy at port 8123

### Expected Behavior

HTTP proxy running in port 8123

### Actual Behavior

Error message: Some AdGuard services were not started correctly. Please contact the support team.

Proxy server not started, error in console: 
```
E: (CoreLibServiceHandler: restart()) Can't start proxy server. Error: Error Domain=ANServiceManager Code=1 "The service is already running or being started" UserInfo={NSLocalizedDescription=The service is already running or being started}
```

### Screenshots

<details><summary>Screenshot 1</summary>

</details>

### Additional Information

Tried uninstalling (including network extension) and reinstall, no luck.

**Comment by dingwen07** [comment]
The HTTP Proxy works (and AdGuard won't prompt error) if I have `Automatically filter applications` enabled, either `Automatic Proxy` mode or `Network Extension` mode.

Some more error log:

```
E: (CoreLibServiceHandler: restartProxy()) Can't start proxy server. First attempt. Error: Error Domain=ANProxyServerErrorDomain Code=6 "Can't apply kext rules or config" UserInfo={NSLocalizedDescription=Can't apply kext rules or config}

E: (CoreLibServiceHandler: restart()) Can't start proxy server. Error: Error Domain=ANProxyServerErrorDomain Code=6 "Can't apply kext rules or config" UserInfo={NSLocalizedDescription=Can't apply kext rules or config}
```

**Comment by AlexandrPkhm** [comment]
Hi @dingwen07 

We have recently released a new version of AdGuard for Mac 2.15. Could you please try updating the application and see if the issue persists? Please let us know the results.

**Comment by dingwen07** [comment]
> Hi @dingwen07
> 
> We have recently released a new version of AdGuard for Mac 2.15. Could you please try updating the application and see if the issue persists? Please let us know the results.

Issue resolved in AdGuard for Mac 2.15.
